core/admin.py

```diff
@@ -199,6 +200,12 @@ def view_add_organizers(self, request):
             'form': form,
             'title': 'Add organizers',
         })
+    
+    def save_model(self, request, obj, form, change):
+        created = not obj.pk
+        super(EventAdmin, self).save_model(request, obj, form, change)
+        if created:
+            form.add_default_data(obj)
```

Comment:
`ModelAdmin` calls all form saves with `commit=False` as it expects the instance to be saved later using `save_model`.